rspec-mode.el

```diff
@@ -116,11 +116,11 @@
   "RSpec minor mode."
   :group 'languages)
 
-(defcustom rspec-use-rake-when-possible t
+(defcustom rspec-use-rake-when-possible nil
   "When non-nil and Rakefile is present, run specs via rake spec task."
   :tag "RSpec runner command"
-  :type '(radio (const :tag "Use 'rake spec' task" t)
-                (const :tag "Use 'spec' command" nil))
+  :type '(radio (const :tag "Use 'spec' command" nil)
+		(const :tag "Use 'rake spec' task" t))
```

Comment:
Indentation fixed. 

I swapped the options because it seems to me that the default option should be first. I'll leave it this way if you don't mind.